deploy/olm-catalog/akka-cluster-operator/1.0.0/akka-cluster-operator-certified.v1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,327 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: akka-cluster-operator-certified.v1.0.0
+  namespace: placeholder
+  annotations:
+    capabilities: Seamless Upgrades
+    categories: Application Runtime
+    certified: 'true'
+    containerImage: 'registry.connect.redhat.com/lightbend/akka-cluster-operator-certified:0.2.2'
```

Comment:
And this 1.0.0?

Reply:
Yes, if there was a 1.0.1 image, but it got eaten by the verification tooling issue and I ended up w/ 1.0.1 as my tag. Could push a 1.0.0 tag image to keep things clean. Can also run a metadata only 1.0.1 release and immediately follow up w/ a 1.0.2 to get things tidy again.